readthedocsext/theme/templates/projects/partials/dashboard/example-projects.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+
+{% with "https://docs.readthedocs.io/page/examples.html" as url %}
+  <div class="ui card">
+    <div class="content">
+      <a class="header" target="_blank" href="{{ url }}">{% trans "Get inspiration from example projects" %}</a>
+      <div class="description">
+        {% blocktrans %}
+          Not sure how to get started?
+          Check out these example projects for a few common tools to get some inspiration.
+        {% endblocktrans %}
+      </div>
+    </div>
+    <div class="ui grid extra content">
+      <a class="ui sixteen wide column positive basic button" target="_blank" href="{{ url }}">
+        {% trans "Check out example projects" %}
+      </a>
+    </div>
```

Comment:
These don't need grids to affect the content width of the button and can instead use a `fluid` button. I would use `primary` instead of `positive` too. The card examples show `positive`/`negative` as an example of a yes/no dialog prompt, where instead we just want a nice looking link.

And the button looks quite big, I might try `small` to reduce it just a little.

```suggestion
    <div class="ui extra content">
      <a class="ui small primary fluid basic button" target="_blank" href="{{ url }}">
        {% trans "Check out example projects" %}
      </a>
    </div>
```

Reply:
This is how it looks with this change:

![Screenshot_2024-04-19_17-48-31](https://github.com/readthedocs/ext-theme/assets/244656/aa375881-f409-459e-bb82-fe24c5e7f6ef)

I think it's fine 👍🏼 

> These don't need grids to affect the content width of the button and can instead use a `fluid` button

Honestly, I'm not able to understand how this pattern works yet and I had pretty bad time trying out these different classes 😅 until I found something that didn't break. I need to go back to the basics of this framework at some point and try to understand it.

Reply:
The changes you have here look like what I was describing, yeah. But the code in the PR is still not updated it seems.